editors/emacs/Portfile

```diff
@@ -103,7 +103,7 @@ platform darwin {
 
 if {$subport eq $name || $subport eq "emacs-app"} {
     version         29.1
-    revision        2
+    revision        0
```

Comment:
Is this actually correct?

Reply:
@pmetzger it is side effect of port bump; I’ll take care of that in hour or two